app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,40 @@
+class CommentsController < ApplicationController
+
+  def new
+    @comment = Comment.new
+  end
+
+  def create
+    @comment = Comment.new(comment_params)
+    if @comment.save
+      render :_show
+    else
+      render :new
+    end
+  end
+
+  def edit
+    @comment = Comment.find(params[:comment].id)
```

Comment:
Don't need this after all for comments since we're only having create and destroy but I'll keep it in mind for future controllers.